qa/common/util.sh

```diff
@@ -392,3 +392,45 @@ function check_test_results () {
 
     return 0
 }
+
+# Run three inference servers and return immediately. Sets pid for each server
+# correspondingly, or 0 if error.
+function run_multiple_servers_nowait () {
+    if [ -z "$SERVER" ]; then
+        echo "=== SERVER must be defined"
+        return
+    fi
+
+    if [ ! -f "$SERVER" ]; then
+        echo "=== $SERVER does not exist"
+        return
+    fi
+
+    SERVER1_PID=0
+    SERVER2_PID=0
+    SERVER3_PID=0
```

Comment:
can you make this helper function dynamic

Reply:
Done